### ISSUE TITLE
Question about training from pretrained model for reranking

### ISSUE THREAD

**Issue by xjh19971** [bytedance/R2Former#6]
Hi, I noticed that in Line 2 of [train_reranking.sh](https://github.com/bytedance/R2Former/blob/main/train_reranking.sh), the model does not load the weights trained in the global retrieval part. Does it mean that the reranking model is trained by the pre-trained weight of the backbone? Thank you!

**Comment by Jeff-Zilence** [comment]
You may check the --resume. It loads the model.

**Comment by xjh19971** [comment]
Thanks for your reply, but I mean Line 2 of train_reranking.sh does not have --resume, which means it does not load the global retrieval model weights as I understand. Do I misunderstand something?

**Comment by LKELN** [comment]
Is that the size of the dataset you're useing  @xjh19971 
![b4fa6c8c1a2ee263b6a5da88903f579](https://github.com/bytedance/R2Former/assets/109862533/4f0105e6-c764-4033-8933-261d04de7a2e)

**Comment by xjh19971** [comment]
> Is that the size of the dataset you're useing @xjh19971 ![b4fa6c8c1a2ee263b6a5da88903f579](https://user-images.githubusercontent.com/109862533/268448586-4f0105e6-c764-4033-8933-261d04de7a2e.png)

@LKELN Yes

**Comment by LKELN** [comment]
I have followed the training configuration provided by the author twice for training, but the accuracy now does not meet the accuracy provided by the author in  paper.the first time in A800 R@1: 87.4, R@5: 93.8, R@10: 94.5, R@20: 95.4, R@100: 97.0,the second time in RTX3090 R@1: 84.5, R@5: 91.5, R@10: 93.2, R@20. 93.8, R@100: 95.5 What is the accuracy you have trained? Did the author use data augmentation when training the backbone? @xjh19971

**Comment by Jeff-Zilence** [comment]
> Thanks for your reply, but I mean Line 2 of train_reranking.sh does not have --resume, which means it does not load the global retrieval model weights as I understand. Do I misunderstand something?

Yes, it has the --resume command. Please check the script file carefully.

**Comment by xjh19971** [comment]
@Jeff-Zilence thanks for your answer!

**Comment by xjh19971** [comment]
@LKELN I get a similar performance with you but have not fine-tuned (the second command in train_reranking.sh).
